dvc/fs/repo.py

```diff
@@ -366,8 +371,8 @@ def walk(
 
         yield from self._walk(repo_walk, dvc_walk, dvcfiles=dvcfiles)
 
-    def walk_files(self, top, **kwargs):  # pylint: disable=arguments-differ
-        for root, _, files in self.walk(top, **kwargs):
+    def walk_files(self, path_info, **kwargs):
+        for root, _, files in self.walk(path_info):
```

Comment:
looks like we've lost `kwargs`. Is it intentiopnal?

Reply:
Just forgot it.